db/migrate/20230316102305_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :Title
+      t.text :Text
+      t.integer :CommentsCounter
+      t.integer :LikesCounter
+      t.references :user, null: false, foreign_key: true
+
```

Comment:
- [ ] Please make sure to follow the given diagram, as the reference to the users table should be called `author` instead of `user`.